pkg/models/base.go

```diff
@@ -8,8 +8,8 @@ import (
 
 // Model is a basic GoLang struct based on gorm.Model with the JSON tags for openapi3gen
 type Model struct {
-	ID        uint           `gorm:"primarykey" json:"ID,omitempty"`
-	CreatedAt time.Time      `json:"CreatedAt,omitempty"`
-	UpdatedAt time.Time      `json:"UpdatedAt,omitempty"`
-	DeletedAt gorm.DeletedAt `gorm:"index" json:"DeletedAt,omitempty"`
+	ID        uint            `gorm:"primarykey" json:"ID,omitempty"`
+	CreatedAt *time.Time      `json:"CreatedAt,omitempty"`
+	UpdatedAt *time.Time      `json:"UpdatedAt,omitempty"`
+	DeletedAt *gorm.DeletedAt `gorm:"index" json:"DeletedAt,omitempty"`
```

Comment:
```suggestion
	DeletedAt gorm.DeletedAt `gorm:"index" json:"DeletedAt,omitempty"`
```
This and `fmt` solves the CI.

Reply:
My actual suggestion is not using pointers on the DB at all.